api/app/attendance/api.py

```diff
@@ -85,8 +110,48 @@ def post(self):
                 event_name=event.name)
         )
 
-        return attendance, 201
-
+        # Other Fields
+        offer = db.session.query(Offer).filter(
+            Offer.user_id == 1).filter(Offer.event_id == event_id).first()
+
+        if not offer:
+            # Check if Guest Registration
+            registration = None
+            registration_form = db.session.query(RegistrationForm).filter(
+                RegistrationForm.event_id == event_id).first()
+            if(registration_form):
+                registration = db.session.query(GuestRegistration).filter(
+                    GuestRegistration.user_id == user_id).filter(GuestRegistration.registration_form_id == registration_form.id).first()
+
+            if not registration:
+                return OFFER_NOT_FOUND
+        else:
+            # Normal registration
+            registration = db.session.query(Registration).filter(

```

Comment:
Some people may have not filled in their registration form, but we still want to be able to check them in - rather just return blanks or unknowns or something for the fields that come from the registration form, rather than an error.

Reply:
Done that.